test-integ/peering_commontopo/main_test.go

```diff
@@ -60,11 +52,8 @@ func TestAllCommonTopoSuites(t *testing.T) {
 	}
 }
 
-func TestJustCommonTopo(t *testing.T) {
```

Comment:
Not sure why you're deleting this? While it is implied by the other tests, it's nice to be able to test the basics by itself.

Reply:
Wasn't sure what it was used for. Didn't see it referenced anywhere